test/test_unary_ufuncs.py

```diff
@@ -486,34 +489,6 @@ def test_batch_vs_slicing(self, device, dtype, op):
 
         self.assertEqual(actual, expected)
 
-    def _test_out_arg(self, op, input, output, expected, **kwargs):
-        if op.safe_casts_outputs:
-            expect_fail = not torch.can_cast(expected.dtype, output.dtype)
-        else:
-            expect_fail = output.dtype != expected.dtype
-
-        if expect_fail:
-            with self.assertRaises(RuntimeError):
-                op(input, out=output, **kwargs)
-        else:
-            res = op(input, out=output, **kwargs)
-            self.assertTrue(res is output)
-            self.assertEqual(output, expected.to(output.dtype))
-
-    @ops(unary_ufuncs, dtypes=OpDTypes.supported)
-    def test_out_arg_all_dtypes(self, device, dtype, op):
```

Comment:
is this checked in test_out? Not for all dtypes, I'm sure, but that's probably fine

Reply:
We don't check safe casting to a higher dtype in test_out yet, no